launcher/cog-launcher.c

```diff
@@ -373,6 +373,12 @@ cog_launcher_startup(GApplication *application)
 #endif
 }
 
+static void
+cog_launcher_activate(GApplication *application)
+{
+    G_APPLICATION_CLASS(cog_launcher_parent_class)->activate(application);
```

Comment:
I would add a comment here saying that we handle this just to avoid a warning but in practice we do nothing so far